Extension/package.nls.json

```diff
@@ -951,6 +951,33 @@
     "c_cpp.debuggers.vsCodeCommand.command.description": "VS Code command to be invoked.",
     "c_cpp.debuggers.vsCodeCommand.args.description": "Arguments to the VS Code command.",
+    "c_cpp.debuggers.cpplldb.debuggerPath.description": "Full path to the LLDB-DAP debugger executable. If not specified, the extension will search for the debugger in the system PATH.",
+    "c_cpp.debuggers.cpplldb.debuggerArgs.description": "Additional arguments for the LLDB-DAP debugger.",
+    "c_cpp.debuggers.cpplldb.disableaslr.description": "Disable Address Space Layout Randomization (if supported).",
+    "c_cpp.debuggers.cpplldb.disablestdio.description": "Disables the standard input/output streams during debugging.",
+    "c_cpp.debuggers.cpplldb.expandShellArguments.description": "Expand program arguments as a shell would (does not actually launch the program in a shell).",
+    "c_cpp.debuggers.cpplldb.detachOnError.description": "Detach the debugger when an error occurs. ",
+    "c_cpp.debuggers.cpplldb.debuggerRoot.description": "The directory to where relative object files can be located.",
+    "c_cpp.debuggers.cpplldb.targetTriple.description": "Overrides the triplet of the target program",
+    "c_cpp.debuggers.cpplldb.platformName.description": "Overrides the platform name of the target program",
+    "c_cpp.debuggers.cpplldb.initCommands.description": "LLDB commands executed upon debugger startup prior to creating the LLDB target.",
+    "c_cpp.debuggers.cpplldb.preRunCommands.description": "LLDB commands executed just before launching/attaching, after the LLDB target has been created.",
+    "c_cpp.debuggers.cpplldb.postRunCommands.description": "LLDB commands executed after launching when it's in a stopped state prior to any automatic continuation.",
+    "c_cpp.debuggers.cpplldb.launchCommands.description": "Custom LLDB commands that are executed instead of launching a process.",
+    "c_cpp.debuggers.cpplldb.stopCommands.description": "LLDB commands executed each time the program stops.",
+    "c_cpp.debuggers.cpplldb.exitCommands.description": "LLDB ommands executed when the program exits.",
+    "c_cpp.debuggers.cpplldb.terminateCommands.description": "LLDB commands executed when the debugging session ends.",
+    "c_cpp.debuggers.cpplldb.enableAutoVariableSummaries.description": "Enable auto generated summaries for variables when no summaries exist for a given type. This feature can cause performance delays in large projects when viewing variables.",
+    "c_cpp.debuggers.cpplldb.displayExtendedBacktrace.description": "Enable language specific extended backtraces.",
+    "c_cpp.debuggers.cpplldb.enableSyntheticChildDebugging.description": "If a variable is displayed using a synthetic children, also display the actual contents of the variable at the end under an entry. This is useful when creating sythetic child plug-ins as it lets you see the actual contents of the variable.",
```

Comment:
It seems like `a synthetic children` should be something that matches in either singular or plural like "synthetic children" or "a synthetic child", whichever is more accurate.